server.go

```diff
@@ -208,6 +209,19 @@ func StatsHandler(h stats.Handler) ServerOption {
 	}
 }
 
+// UnknownStreamHandler returns a ServerOption that allows for adding a custom handler for unknown gRPC services/methods.
```

Comment:
And in the comment, we should mention how the custom handler will be used (and that it's bidi streaming).
All unknown gRPC services/methods will be handled by this handler as bidirectional streaming RPCs.